src/components/commitmentRenewal/CommitmentRenewal.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright 2025 SAP SE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import {
+  Button,
+  DataGrid,
+  DataGridRow,
+  DataGridCell,
+  DataGridHeadCell,
+  Select,
+  SelectOption,
+  Stack,
+  Message,
+} from "@cloudoperators/juno-ui-components/index";
+import RenewModal from "../commitment/Modals/RenewModal";
+import { t, formatTimeISO8160 } from "../../lib/utils";
+import { Unit, valueWithUnit } from "../../lib/unit";
+import { categoryTitle } from "../paygAvailability/stylescss";
+import { useMutation } from "@tanstack/react-query";
+import { createCommitmentStoreActions } from "../StoreProvider";
+
+export const renewableInfoText = "No renewable commitments found for this project.";
+export const renewableInfoHint = [
+  "You will usually not need to check this page proactively.",
+  "We will send mail notifications when active commitments are about to expire.",
```

Comment:
```suggestion
  "We will send mail notifications when your active commitments are nearing expiration.",
```

Reply:
Adding "your" is a good point. "Nearing expiration" does not seem like an improvement to me. Usually, any shorter phrasing that uses less helper verbs is going to be clearer.

Reply:
fine for me :)